packages/bridge/src/type-definition/FieldTypeEnum.ts

```diff
@@ -1,5 +1,8 @@
 export enum FieldTypeEnum {
   string = 'string',
   number = 'number',
-  uint64_t = 'uint64_t',
+  int = 'int',
+  bool = 'bool',
+  address = 'address',
+  double = 'double'
```

Comment:
- **double** and number have the same value? Should be allowed only one
- use **account** instead of **address**.

Reply:
some javascript developer don't know about `double`, they just want to create simple smart contract, but some c++ developer they know much about uint64, uint32, uint16, so I added all of their type.